examples/bert/README.md

```diff
@@ -126,7 +127,12 @@ The `create_vocabulary.py` script allows you to compute your own WordPiece
 vocabulary for use with BERT. In most cases however, it is desirable to use the
 standard BERT vocabularies from the original models. You can download the
 English uncased vocabulary
-[here](https://storage.googleapis.com/tensorflow/keras-nlp/examples/bert/bert_vocab_uncased.txt).
+[here](https://storage.googleapis.com/tensorflow/keras-nlp/examples/bert/bert_vocab_uncased.txt),
+or in your terminal run:
+```shell
+curl https://storage.googleapis.com/tensorflow/keras-nlp/examples/bert/bert_vocab_uncased.txt \
```

Comment:
same here `curl -O url` to keep this a one liner.

Reply:
I am doing a renaming here - in the README we are mixing vocab.txt with bert_vocab_uncased.txt, so to unify them I am only keeping vocab.txt at every reference.